nilearn/decomposition/tests/test_canica.py

```diff
@@ -114,3 +107,7 @@ def test_component_sign():
                 canica.components_)):
             mp = mp.get_data()
             assert_less_equal(-mp.min(), mp.max())
+
+
+if __name__ == '__main__':
+    test_component_sign()
```

Comment:
We tend to run tests using nose from the command line. Thus I think that we should remove this line.